src/server/twitterSearch.js

```diff
@@ -25,10 +25,10 @@ export const TwitAccess = new Twit({
  */
 const buildTweetFromRaw = (rawTweet) => {
   return Builders.TweetBuilder()
-    .id(rawTweet.id)
+    .id(rawTweet.id_str)

```

Comment:
Yep very important we do NOT use id, they're slightly different and id_str is the one that actually links to the tweet